radix-engine/src/system/transaction/multithread_intent_processor.rs

```diff
@@ -138,6 +196,47 @@ impl MultiThreadIntentProcessor {
         Ok(())
     }
 
+    fn create_temp_auth_zone<Y: SystemBasedKernelApi>(api: &mut Y) -> Result<NodeId, RuntimeError> {
```

Comment:
MINOR: If we keep this, from a code clarity perspective, I'd ideally like to explain what this does more clearly, e.g.

```rust
fn create_child_auth_zone_with_transaction_processor_global_caller(...)
```

Reply:
Okay, after having it explained, perhaps we could call it:
```rust
fn create_temporary_child_auth_zone_for_verify_parent(api: impl SystemBasedKernelApi) -> Result<NodeId, RuntimeError> {
//..
}
```